src/import-data/ImportDataDestination.ts

```diff
@@ -220,14 +219,20 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
             }),
           ]),
       ]),
-      importMayTakeTime && div({ style: { marginTop: '0.5rem', lineHeight: '1.5' } }, [importMayTakeTimeMessage]),
-      !!selectedWorkspace &&
-        h(Fragment, [
-          h(WorkspacePolicies, { workspace: selectedWorkspace }),
-          !!isProtectedData &&
-            isAzureWorkspace(selectedWorkspace) &&
-            div([p(['Importing this data may add:']), ul([li(['Additional access controls'])])]),
+      importMayTakeTime &&
```

Comment:
Screenshots of this section.

No policies:
![image](https://github.com/DataBiosphere/terra-ui/assets/484484/5e6092bd-762d-487c-9ba6-2b0e20908a70)

Policies:
![image](https://github.com/DataBiosphere/terra-ui/assets/484484/e096f14b-d115-43f5-a54c-0baa7da47070)

Protected data with and without policies: